### PR TITLE
 Query multiple tables

### PR DESCRIPTION
**Pull request by Estherstarr**
- [ ]  Create a table named owners with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
full_name: string
age: integer
Screenshot 2022-07-09 at 4 42 58 PM

- [ ] Create a table named species with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
Screenshot 2022-07-09 at 4 44 48 PM

- [ ] Modify animals table:
Make sure that id is set as autoincremented PRIMARY KEY
Remove column species
Add column species_id which is a foreign key referencing species table
Add column owner_id which is a foreign key referencing the owners table
Screenshot 2022-07-09 at 4 46 48 PM

- [ ] Insert the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.
Screenshot 2022-07-09 at 4 50 38 PM
Screenshot 2022-07-09 at 5 08 44 PM

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+ALTER TABLE animals
+ADD species_id INT;
+ALTER TABLE animals
+ADD CONSTRAINT fk_species_table FOREIGN KEY(species_id) REFERENCES species(id);
+
+-- Add column owner_id which is a foreign key referencing the owners table
+ALTER TABLE animals
+ADD owner_id INT;
+ALTER TABLE animals
+ADD CONSTRAINT fk_owner_table FOREIGN KEY(owner_id) REFERENCES owners(id);
```
Comment (zhadier) [inline schema.sql]: - [ ] Great job making the queries, however you seem to have misunderstood the tasks. You needed to create two completely new tables called `owners` and `species` here, and not add the entities to the `animals` table directly. Then you can create foreign keys that reference the species and the owner id's inside the `animals` table. Kindly add these two tables, alter the queries so they work with the new tables and resubmit the project. I just saw that you added them inside the PR description but you also need to add them inside the files 
For example, the owners table can be created like below
``` SQL
CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );

```

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Estherstarr  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ GitFlow was set up correctly
✔️ Professional pull request title and description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
- [ ] Create tables appropriately

**Review by danmainah — APPROVED** [review]
Hi @Estherstarr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — COMMENTED** [review]
Hi @Estherstarr , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You seem to have submitted a project that is for the wrong Milestone. Kindly add the correct PR link in the dashboard so we can proceed to review it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**